.github/workflows/helm-ci.yml

```
@@ -18,11 +18,14 @@ jobs:
   conftest:
     runs-on: ubuntu-latest
     container:
-      image: grafana/mimir-build-image:pr3976-e7cae18e3
+      image: grafana/mimir-build-image:goupdate-751733fe1
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
 
+      - name: Run Git Config
+        run: git config --global --add safe.directory /__w/mimir/mimir
```

```suggestion
        run: git config --global --add safe.directory '*'
```

We can use this version and not rely on specific placement of Mimir repository. (We could use `$GITHUB_WORKSPACE` too)

Applies to all these lines.